.github/workflows/cross.yml

```
@@ -63,3 +63,19 @@ jobs:
             NDK_PROJECT_PATH=. \
             APP_BUILD_SCRIPT=jni/Android.mk \
             NDK_APPLICATION_MK=jni/Application.mk
+
+  os2-emx:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+      - name: Build
+        uses: komh/setup-cross-os2emx@v1
```

actionlint (1.7.12)

[error] unexpected key "uses" for step to run shell command. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "working-directory"

(syntax-check)